README.es.md

```diff
@@ -0,0 +1,310 @@
+![GitHub - Top language](https://img.shields.io/github/languages/top/onfleet/node-onfleet.svg?style=popout-square)
+[![npm - Downloads](https://img.shields.io/npm/dt/@onfleet/node-onfleet.svg?style=popout-square)](https://www.npmjs.com/package/@onfleet/node-onfleet)
+
+> *Consulta este documento en otro idioma*:  
+> [English](https://github.com/onfleet/node-onfleet/blob/master/README.md)  
+> [Français](https://github.com/onfleet/node-onfleet/blob/master/README.fr.md)  
+> [正體中文](https://github.com/onfleet/node-onfleet/blob/master/README.zh-tw.md)  
+
+Los invitamos a visitar nuestra publicación sobre el [proyecto de librerías para la API](https://onfleet.com/blog/api-wrappers-explained/) para conocer más sobre nuestras iniciativas.  
+En caso de preguntas, pueden contactarnos a través de un issue [aquí](https://github.com/onfleet/pyonfleet/issues) o escribirnos a support@onfleet.com.
+
+## Tabla de contenidos
+* [Table of contents](#table-of-contents)
+* [Sinopsis](#sinopsis)
+* [Instalación](#instalación)
+* [Uso](#uso)
+    - [Autenticación](#autenticación)
+    - [Pruebas unitarias](#pruebas-unitarias)
+    - [Límites](#límites)
+    - [Respuestas](#respuestas)
+    - [Operaciones CRUD soportadas](#operaciones-crud-soportadas)
+        * [Peticiones GET](#peticiones-get)
+            - [Ejemplos de `get()`](#ejemplos-de-get)
+            - [Ejemplos de `get(parametro)`](#ejemplos-de-getparametro)
+            - [Ejemplos de `getByLocation`](#ejemplos-de-getbylocation)
+        * [Peticiones POST](#peticiones-post)
+            - [Ejemplos de `create()`](#ejemplos-de-create)
+        * [Peticiones PUT](#peticiones-put)
+            - [Ejemplos de `update()`](#ejemplos-de-update)
+            - [Ejemplos de `insertTask()`](#ejemplos-de-inserttask)
+        * [Peticiones DELETE](#peticiones-delete)
+            - [Ejemplos de `deleteOne()`](#ejemplos-de-deleteone)
+    - [Ejemplos de cómo utilizar las operaciones CRUD](#ejemplos-de-como-utilizar-las-operaciones-crud)
+    - [Qué NO hacer](#que-no-hacer)
+
+## Sinopsis
+La librería en Node.js de Onfleet nos permite un acceso fácil y cómodo a la API de Onfleet.
+
+## Instalación
+```
+npm install @onfleet/node-onfleet
+```
+Para TypeScript, debemos instalar la [definición tipada](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/onfleet__node-onfleet):
+```
+npm install @types/onfleet__node-onfleet
+```
+> ¡Agradecimientos especiales a @marcobeltempo por la contribución!
+
+## Uso
+Antes de usar la librería, es indispensable obtener una llave para la API a través de alguno de los administradores de la organización a la que pertenecemos.
+
+La creación e integración de llaves se realiza a través del [panel principal de Onfleet](https://onfleet.com/dashboard#/manage).
+
+Para utilizar la librería solo tenemos que crear uns instancia de `Onfleet` usando la llave:
```

Comment:
`solo` Vs. `sólo`

Not gonna fight you over this, but I'm on team `sólo` 😋
The RAE's rules make no sense at all.

Reply:
I'm also on that team, this was not intended 😄